cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -158,3 +158,5 @@ spec:
   disruptionManagement: #The section for configuring management of daemon disruptions
     managePodBudgets: false #if `true`, the operator will create and manage PodDsruptionBudgets for OSD, Mon, RGW, and MDS daemons. OSD PDBs are managed dynamically via the strategy outlined in the [design](https://github.com/rook/rook/blob/master/design/ceph-managed-disruptionbudgets.md). The operator will block eviction of OSDs by default and unblock them safely when drains are detected.
     osdMaintenanceTimeout: 30 # is a duration in minutes that determines how long an entire failureDomain like `region/zone/host` will be held in `noout` (in addition to the default DOWN/OUT interval) when it is draining. This is only relevant when  `managePodBudgets` is `true`. The default value is `30` minutes.
+    manageMachineDisruptionBudgets: true
```

Comment:
can you also add a disruptionManagement section to cluster-on-pvc.yaml and change managePodBudgets to true over there?

Reply:
`manageMachineDisruptionBudgets` will need to be `false` except for openshift deployments. Today we don't have an openshift example for cluster.yaml, so I'd suggest a comment here that it's recommended to change this to `true` on openshift.

Reply:
@rohantmp What if we change `managePodBudgets` to `true` in the examples? Or is there a reason to disable it by default? Note that upgraded clusters would still have the default of `false`, assuming that is the default when the property isn't found.

Reply:
Missed this. We could do that, but it would break in some scenarios. Eg: OSDs on PVs with failureDomain host would result in nodes being undrainable as the PVC name is being used as the host name IIRC.

Reply:
Good point, then let's leave it as false. Do we need to add more comments to that setting so users don't enable it in scenarios like this?

Reply:
yeah, I'll submit a PR for that soon.